components/resource_control/src/service.rs

```diff
@@ -139,6 +151,180 @@ impl ResourceManagerService {
             }
         }
     }
+
+    async fn load_controller_config(&self) -> Result<RequestUnitConfig, ()> {
```

Comment:
I don't see when this function will return Err, remove the Result wrapper type

Reply:
Okay, it was modified to option. I think when meeting error needs to fallback to the default config other than check it regularly.